library/public/image.h

```diff
@@ -100,7 +100,7 @@ class F3D_EXPORT image
    * Save an image to a file in .png format
    * Can throw an image::exception if image is not sane.
    */
-  image& save(const std::string& path);
+  const image& save(const std::string& path) const;
```

Comment:
I think a global constness pass on the API should be done

Reply:
out of scope ofc

Reply:
I agree